database/migrations/2017_07_28_175515_create_order_items_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOrderItemsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_items', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('target_id')->unsigned();
+            $table->string('target_type');
```

Comment:
-you should use $table->morphs('targetable') instead of $table->integer('target_id')->unsigned() and $table->string('target_type').